### PR TITLE
Looping videos show loading spinner for a split second before looping

### PR DESCRIPTION
**Pull request by MariaAlmeidaMJ**

### CODE REVIEW

**Review by leroybm — APPROVED** [review]
LGTM code wise

**Review by fpfcarvalho — APPROVED** [review]

File: src/modules/adsupport.js [hunk 1]
```diff
@@ -3,7 +3,7 @@ export default function (playerInstance, options) {
     const VPAID_VERSION = '2.0';
 
     playerInstance.renderLinearAd = (adListId, backupTheVideoTime) => {
-        playerInstance.toggleLoader(true);
+        playerInstance.toggleLoader(false);
```
Comment (golikov) [inline src/modules/adsupport.js:6]: Won't this have some undesired impacts? Like not showing the loader while ad is being loaded? Pretty sure the loader there is intentional..

Comment (MariaAlmeidaMJ) [reply src/modules/adsupport.js:6]: @golikov With this, the loader doesn't appear at the same time as the play button. I've tested it with the slow net and I see the loader popping up when it's loading more content

Comment (golikov) [reply src/modules/adsupport.js:6]: I am not going to be testing that just to prove a point, I still don't believe that won't break a loder where we kind of expect it, so let's hope the QA catches that.

**Review by golikov — APPROVED** [review]

**Pull request merged**